### DISCUSSION TITLE
How i can print the probably of NER results

### DISCUSSION THREAD

**Discussion by buvata** [explosion/spaCy#10700] [Help: Best practices]
I want to know of the probably of the predict's model Spacy. Can u help me ? 
![Screenshot from 2022-04-25 14-14-24](https://user-images.githubusercontent.com/42181609/165038738-4eca8931-50ce-46c3-a098-77a110277603.png)

**Comment by thomashacker** [comment]
Hello, it's not possible to show the confidence scores. You can read more about this [here](https://github.com/explosion/spaCy/discussions/9189#discussioncomment-1311602)
